EMpy/modesolvers/FD.py

```diff
@@ -329,17 +336,17 @@ def __init__(self, wl, x, y, epsfunc, boundary):
     def _get_eps(self, xc, yc):
         tmp = self.epsfunc(xc, yc)
 
-        def _reshape(tmp):
+        def _reshape(tmp): # pad the array by duplicating edge values
```

Comment:
This is not PEP8 compliant. I know that at the moment EMpy is not very PEP8 compliant at all, but new code should be.
Can you turn this comment into a docstring instead?